integration_tests/e2e/goal/addNote.cy.ts

```diff
@@ -44,7 +44,7 @@ context('Add a note', () => {
     const createGoalPage = overviewPage.clickAddGoalButton()
     createGoalPage //
       .setGoalTitle('Learn French')
-      .setGoalTargetDate()
+      .setTargetCompletionDate0to3Months()
```

Comment:
Not strictly related to Update Goal, but I renamed this page method on `CreateGoalPage` to better reflect what it does (the thinking being we might want to add other methods one day that select the 3-6 month option, or the 6-12 month option etc